### ISSUE TITLE
AutoSearchInput component does not have a valid label

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons#12560]
The `AutoSearchInput` does not have a valid label according to the WAVE a11y tool because there is no link between the `input` and the `label`. That's because we do not use the `id` attribute but a `name` attribute to link both HTML tags.
